recipe/meta.yaml

```diff
@@ -19,67 +19,130 @@
 {% endif %}
 
 package:
-  name: {{ name | lower }}
+  name: {{ name | lower }}-suite
   version: {{ version }}
 
 source:
-  url: https://github.com/grpc/grpc/archive/v{{ version }}.tar.gz
-  sha256: 291db3c4e030164421b89833ee761a2e6ca06b1d1f8e67953df762665d89439d
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: bdb8e98145469d58c69ab9f2c9e0bd838c2836a99b5760bc0ebf658623768f52
   patches:
-    - force-protoc-executable.patch
-  # You can use module version of re2 if you add this and chage build.sh:
-  # - git_url: https://github.com/google/re2
-  #   git_ref: aecba11114cf1fac5497aeb844b6966106de3eb6
-  #   folder: third_party/re2
+    - patches/force-protoc-executable.patch
+    - patches/0001-Monkey-patch-distutils.ccompiler.spawn-to-elide-std-.patch
+    - patches/0002-windows-ssl-lib-names.patch
+    - patches/0001-fix-win-commands.patch
+    - patches/0003-Link-against-grpc.patch
 
 build:
-  number: 1
+  number: 0
+
+outputs:
+  - name: {{ name }}-cpp
+    script: build-cpp.sh   # [not win]
+    script: build-cpp.bat  # [win]
+    build:
+      missing_dso_whitelist:
+        - '$RPATH/ld64.so.1'  # [s390x]
+      script_env:
+        - STATIC_BUILD={{ static_build }}
+        - LIB_EXT={{libext}}
 {% if static_build == 'yes' %}
-  ignore_run_exports:
-    - abseil-cpp
-    - c-ares
-    - re2
-    - openssl
-    - zlib
-{% else %}
-  run_exports:
-    - {{ pin_subpackage('grpc-cpp', max_pin='x.x') }}
+      ignore_run_exports:
+        - abseil-cpp
+        - c-ares
+        - libprotobuf  # libprotobuf pins itself to x.x via run_exports
+        - re2
+        - openssl
+        - zlib
 {% endif %}
```

Comment:
FYI, you cannot trust the link check for static libraries, because lief doesn't support static libraries, and so you get zero information about the required dependencies in the interface of static libs. These dependencies will still have to be present in the same environment as a static grpc build, if you want to link to it.

Here's an [explanation](https://cmake.org/pipermail/cmake/2016-May/063400.html) about this from the cmake mailing list, excerpt:
> [... A depends on B ...] If, however, A is a static library, the situation changes. Static libraries do not carry information about other libraries they depend on. For this reason, when A links in B as PRIVATE and another target C links in A, CMake will still add B to the list of libraries to be linked for C because parts of B are needed by A, but A itself doesn't have that dependency encoded into it. So even though B is an internal implementation detail of A, C still needs B added to the linker command, which CMake conveniently handles for you.